### PR TITLE
Add support for WPML config files

### PR DESCRIPTION
**Pull request by displaynone**
**Changes proposed:**

- [x] Add

Some WordPress Themes and Plugins include an [wpml-config.xml](https://wpml.org/documentation/support/language-configuration-files/) file so it works with WPML

### CODE REVIEW

File: src/iconsManifest/supportedExtensions.ts [hunk 1]
```diff
@@ -3118,6 +3118,16 @@ export const extensions: IFileCollection = {
       extensions: ['doc', 'docx', 'docm', 'dot', 'dotx', 'dotm', 'wll'],
       format: FileFormat.svg,
     },
+    {
+      icon: 'webpack',
```
Comment (JimiC) [inline src/iconsManifest/supportedExtensions.ts]: You do realize that the `icon` name does not conform with the file name of the icon file.

**Comment by displaynone** [comment]
@JimiC where can I find your art guidelines? I couldn't find it

I realized SVG width and height are wrong, but is there any other issue?

**Comment by JimiC** [comment]
The art guidelines are scarred here and there I'm afraid. But by observing the existing icons you can get a general idea.

**Comment by displaynone** [comment]
Thanks @jimic, crazy last weeks and I wasn't able to fix it

**Comment by JimiC** [comment]
NP

**Pull request merged**